lib/gui/models/drives.js

```diff
@@ -20,103 +20,94 @@
  * @module Etcher.Models.Drives
```

Comment:
Ahhhh, thanks Benedict :-)
I don't suppose there's any (easy) way to have an 'abstract' `store.js`, that could be plugged into `localStorage` when using Electron, or something-else when using the CLI?